### PR TITLE
Robot skills cleanup

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Merge after #704

### CODE REVIEW

**Comment by LoyVanBeek** [comment]
This PR is about only the last commit, it seems. 
If so, can you ditch this PR, make a new one and cherry-pick that single commit onto that branch? Or do some fancy rebasing? Because this is not review-able this way.

**Comment by MatthijsBurgh** [comment]
Merge after XXX, means that your diff will be much small when that other branch has been merged. So review again @LoyVanBeek, since it has been merge with master again.

**Comment by MatthijsBurgh** [comment]
@LoyVanBeek please review

File: robot_skills/src/robot_skills/ears.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def recognize(self, spec, choices={}, time_out=rospy.Duration(10)):
         if hasattr(self._pre_hook, '__call__'):
             self._pre_hook()
 
-        answer = self._hmi.old_query(spec, choices, timeout=time_out.to_sec())
+        answer = self._hmi.old_query(spec, choices, timeout=time_out.to_sec())  # self._hmi needs to be set in robot
```
Comment (LoyVanBeek) [inline robot_skills/src/robot_skills/ears.py]: Is this still used even?

File: robot_skills/src/robot_skills/world_model_ed.py [hunk 2]
```diff
@@ -377,13 +377,15 @@ def save_image(self, path="", path_suffix="", filename=""):
             f.write(res.json_meta_data)
 
         # rgbd to png
-        os.system('rosrun rgbd rgbd_to_rgb_png %s' % (fname + ".rgbd"))
+        os.system('rosrun rgbd rgbd_to_rgb_png %s' % (fname + ".rgbd"))  # ToDo: very very very ugly
```
Comment (LoyVanBeek) [inline robot_skills/src/robot_skills/world_model_ed.py]: Can you make an issue for this?

File: robot_skills/src/robot_skills/world_model_ed.py [hunk 3]
```diff
@@ -377,13 +377,15 @@ def save_image(self, path="", path_suffix="", filename=""):
...
 
     # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 
     def mesh_entity_in_view(self, id, type=""):
```
Comment (LoyVanBeek) [inline robot_skills/src/robot_skills/world_model_ed.py]: Maybe first mark this as obsolete?

File: robot_skills/src/robot_skills/ears.py (hunk 1 above)
Comment (MatthijsBurgh) [reply robot_skills/src/robot_skills/ears.py]: I don’t think so. But would mean removal of the entire ears. I would do that in a different PR.

File: robot_skills/src/robot_skills/world_model_ed.py (hunk 2 above)
Comment (MatthijsBurgh) [reply robot_skills/src/robot_skills/world_model_ed.py]: #716

File: robot_skills/src/robot_skills/ears.py (hunk 1 above)
Comment (LoyVanBeek) [reply robot_skills/src/robot_skills/ears.py]: :+1:

File: robot_skills/src/robot_skills/world_model_ed.py (hunk 3 above)
Comment (MatthijsBurgh) [reply robot_skills/src/robot_skills/world_model_ed.py]: You mean adding a @deprecated?

Comment (LoyVanBeek) [reply robot_skills/src/robot_skills/world_model_ed.py]: Exactly.

Comment (MatthijsBurgh) [reply robot_skills/src/robot_skills/world_model_ed.py]: Done

**Pull request merged**